make/sources.js

```diff
@@ -36,6 +38,10 @@ exports.lib = [
     "lib/clipper/clipper.js",
 ];
 
+exports.cssrc = [
+    "src/cs/drawgrid.cs",
```

Comment:
Most control flow operations aren't properly supported yet, so chances are you'll have to tweak the transpiler a bit, but in theory yes, it should be fairly easy.